src/ray/constants.h

```diff
@@ -4,6 +4,14 @@
 /// Length of Ray IDs in bytes.
 constexpr int64_t kUniqueIDSize = 20;
 
+/// An ObjectID's bytes are split into the task ID itself and the index of the
+/// object's creation. This is the maximum width of the object index.
+constexpr int kObjectIdIndexSize = 16;
+/// The maximum number of objects that can be returned by a task.
+constexpr int64_t kMaxTaskReturns = (1 << (kObjectIdIndexSize - 1)) - 1;
```

Comment:
It's very difficult to reason about what these constants are, please explain what the values are in the comment (same with the one below).